datm/cime_config/namelist_definition_datm.xml

```diff
@@ -65,7 +68,7 @@
     <type>char</type>
     <category>datm</category>
     <group>datm_nml</group>
-    <valid_values>CLMNCEP,CORE2_NYF,CORE2_IAF,CORE_IAF_JRA,ERA5,CPLHIST</valid_values>
+    <valid_values>CLMNCEP,CORE2_NYF,CORE2_IAF,CORE_IAF_JRA,ERA5,CPLHIST,1PT</valid_values>
```

Comment:
Just an idea but this could be SPOINT or something like that. 1PT did not make sense to me when I saw first time.